sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -221,7 +221,7 @@ export class IdentityClient extends ServiceClient implements INetworkModule {
     controller.signal.onabort = (...params) => {
       this.abortControllers.set(correlationId, undefined);
       if (existingOnAbort) {
-        existingOnAbort(...params);
+        existingOnAbort.apply(controller.signal, params);
```

Comment:
we just create `controller` above, then immediately assign `const existingOnAbort = controller.signal.onabort;`. It feels that `existingOnAbort` would be undefined. But I am not too familiar with this code...

Reply:
You might be correct. I am not sure to be honest - but its all existing code and probably not worth thinking about right now...